docs/file_conventions.rst

```diff
@@ -40,6 +38,10 @@ each of these files).
    deprecated for MAGICC7 in favour of ``*EMIS.IN`` and ``*.SCEN7``
    files.
 
+In future, the MAGICC developers are aiming to move all of MAGICC's input and output
```

Comment:
For future reference it might be useful to add some of the thinking about the new format here ... e.g. describe the blocks.

Reply:
Something like, “this format will always be comprised of 4 sections. The first is a header, for storing whatever text the user wants. This section must always start with ‘—- Header —-‘. It is ignored by MAGICC but can be used by other readers and writers. The next section is other metadata, in key:value pairs. This section must always start with ‘—- Metadata —-‘. The section allows for storage of metadata, like the global attributes section in a netcdf file. This section is also ignored by MAGICC. The third section is a FORTRAN namelist, which stores the flags required to read the file in MAGIC. The flags must match the flags used by MAGICC internally [add link to namelist section below here]. In Pymagicc, these flags are written automatically, the user can not write them. The fourth section is the data. This is always a data block with four header rows: variable, todo, units, region [TODO link to meaning of rows] (although the region row label is ‘Years’). The first column is the time axis and the subsequent columns are the timeseries. This format is highly custom and specialised for use with MAGICC but also to allow human readability. If you want to work with the data, we strongly recommend using Pymagicc’s ‘io’ module to allow easy conversion to python types such as dictionaries, lists, strings and most importantly pandas data frames.”

Reply:
?

Reply:
Great, that clarifies a lot for me!

Is the intention to have something that is "better" specified as previous formats but still readable easily with minimal changes to the Fortran MAGICC readers?

Reply:
In short, yes. Focus a) was definitely on something that was better specified. The fact it's similar to previous stuff is (in my opinion) more luck than design. If it had needed to be completely different, we probably would have changed it. It's just that MAGICC only really has timeseries data like this so the tabular format is fine.